server/views/applications/view.njk

```diff
@@ -118,8 +118,8 @@
             Restricted IPs
           </dt>
           <dd class="govuk-summary-list__value">
-            {% for ip in key.restrictedIPs %}
-              <p>{{ ip }}</p>
+            {% for ipGroup in key.restrictedIPs | batch(1) %}
+                <p>{{ ipGroup | join(", ") }}</p>
             {% endfor %}
```

Comment:
If anyone wants to know what were doing here, Rob and I agreed we could leave this in incase we ever wanted to revisit using multiple columns. Since the batch size is currently 1 this is equivilant to a single column display.